yw_website/yw_website/apps/website/models.py

```diff
@@ -89,7 +89,6 @@ class Meta:
         unique_together = ("data_id", "run")
 
     data_id = models.IntegerField()
-    name = models.TextField()
```

Comment:
why is this being removed?

Reply:
This field is a duplicate

Reply:
it does not look like a duplicate to me

Reply:
ope nevermind its below `in_program_block`